CMakeLists.txt

```diff
@@ -1,5 +1,16 @@
 cmake_minimum_required (VERSION 3.1..3.18)
 
+if(APPLE)
```

Comment:
If I leave out this part, I get the following error:
Bundle identifier is missing. sndfile doesn't have a bundle identifier. Add a value for PRODUCT_BUNDLE_IDENTIFIER in the build settings editor. (in target 'sndfile' from project 'libsndfile')